radar_control/radar_control.py

```diff
@@ -535,6 +535,9 @@ def radar():
     new_experiment_waiting = False
     new_experiment_loaded = True
 
+    # Flag for starting the radar on the minute boundary
+    wait_for_first_scanbound = experiment.slice_dict.get("wait_for_first_scanbound")
```

Comment:
Radical, usually we use experiment.slice_dict['wait_for_first_scanbound']. These two ways of doing it behave differently if the key isn't in the dictionary - dict.get() returns None, whereas dict['nonexistent key'] raises a KeyError. I think for this use case it doesn't really matter since the field defaults, but this could be useful for other parts of the software. Just a comment, nothing to change here.